images/linux/post-generation/homebrew-permissions.sh

```diff
@@ -2,10 +2,13 @@
 
 # Fix permissions for Homebrew
 # https://github.com/actions/virtual-environments/issues/1568
+
 brew_folder="/home/linuxbrew/"
+homebrew_user=$(cut -d: -f1 /etc/passwd | tail -1)
```

Comment:
probably just call it `main_user` or something like this?

Reply:
I guess variables set to "homebrew_user"/"rust_user" to point out that "runner" or "vsts" are actually homebrew/rust users respectively and I hope it should not confuse anyone.